src/dlgPackageExporter.cpp

```diff
@@ -582,3 +800,21 @@ void dlgPackageExporter::listTimers()
         recurseTimers(pChild, pItem);
     }
 }
+
+void dlgPackageExporter::displayResultMessage(const QString& html, bool const isSuccessMessage)
+{
+    if (!isSuccessMessage) {
+        ui->infoLabel->setText(QStringLiteral("<p><font color='red'><b><big>%1</big><b></font></p>").arg(html));
+        return;
+    }
+
+    ui->infoLabel->setText(tr("<p><b><big>%1</big><b></font></p>"
```

Comment:
Please move this out of the text for translation.

Reply:
Agreed - not sure what happened there. :worried:

Reply:
Also the adjacent translation comment needs to be stripped down lots.
It will be mangled in the Crowdin, and is mostly obsolete due to above.

Reply:
Done.